pypolestar/proto/target_soc_service.proto

```
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package pccs.chronos.services.v1;
+
+import "target_soc.proto";
```

Buf (1.67.0)

[error] import "target_soc.proto": file does not exist

(COMPILE)